packages/atclient/include/atclient/monitor.h

```diff
@@ -246,6 +246,11 @@ int atclient_monitor_pkam_authenticate(atclient *monitor_conn, const char *atser
  */
 void atclient_monitor_set_read_timeout(atclient *monitor_conn, const int timeout_ms);
 
+typedef struct atclient_monitor_hooks {
+  int (*pre_decrypt_notification)(void);
```

Comment:
could we typedef this like:

```c
typedef int(atclient_montior_decrypt_notification_hook)(void);
```

Reply:
I originally had done that, but: If the type isn't complicated, I want to avoid typedefs, since if we decide to add parameters to the signature in the future, then we need to make more breaking changes than otherwise necessary.

Reply:
👍🏼 Ok

The reason I brought it up is cause it was a little confusing for me to understand at first

But I also feel like it is okay to keep the breaking change in for now, in order to stay uniform with what we have in `connection_send` as well.